tests/unit/handlers/test_structured_log.py

```diff
@@ -321,3 +321,26 @@ def test_format_overrides(self):
             result = json.loads(handler.format(record))
             for (key, value) in expected_payload.items():
                 self.assertEqual(value, result[key])
+
+    def test_format_with_json_fields(self):
+        """
+        User can add json_fields to the record, which should populate the payload
+        """
+        import logging
+        import json
+
+        handler = self._make_one()
+        message = "name: %s"
+        name_arg = "Daniel"
```

Comment:
question: i am not familiar with the interface, is it supposed to format the message like with print() ?

Reply:
The idea is that Python's logging std lib [allows you to build formated log strings](https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles) like this: `logging.info("my name is: %s", "Daniel")`. This test just makes sure that our GCP library additions don't break that expected functionality